Sample react

paterneluciano1 opened the pull request:
sample react for fedapay demo project

paterneluciano1 committed:
Add files via upload

paterneluciano1 committed:
ad public files

paterneluciano1 committed:
add public files

paterneluciano1 committed:
Update App.js

Saady03 commented on src/App.js:
```diff
@@ -0,0 +1,101 @@
+import { FedaCheckoutButton } from 'fedapay-reactjs';
+
+// Exemple de données produits
+
```
Déplacer la liste de produits vers un fichier json dans un répertoire helpers ou assets

Saady03 commented on src/App.js:
```diff
@@ -0,0 +1,101 @@
+    const { product, publicKey } = this.props;
+    
+    // Configuration dynamique pour le bouton de paiement selon le produit
+
```
Ne pas passer la clé publique via les props. Il vaut mieux montrer les bonnes pratiques en utilisant des variables d'environnement

paterneluciano1 committed:
update README.md

paterneluciano1 committed:
add file .env.example

paterneluciano1 committed:
update file App.js

Modification de la méthode d'implémentation de la clé publique

paterneluciano1 committed:
Update App.js

modification du fichier App.js avec liaison au fichier products.json ou les produits sont importés

paterneluciano1 committed:
Add file products.json

Ajout du fichier products.json ou sont stockés les informations sur les produits

paterneluciano1 committed:
Update App.js
